src/DotLiquid/Properties/Resources.NETCore.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DotLiquid.Properties
+{
+#if NETCore
+    // this class is only used to setup the ResourceManager in Liquid
```

Comment:
ResourceManager in Liquid is based on Type for resource lookup. But because the Resources.Designer.cs is not supported in corefx, I have replaced it with an empty class to help ResourceManager locate the .resources item in the assembly.